docs/setup-dev.md

```diff
@@ -202,3 +202,22 @@ and run tests
 ```
 pytest .
 ```
+
+## Issues with Quick Setup on Ubuntu 16.04
+
+If encountering `fatal error: rocksdb/slice.h: No such file or directory` when running the `init-dev-project.sh <github-name> <new-virtualenv-name>` command during the quick setup (step 7) here is a guide to solve this issue:
+
+Independently install rocksdb with the following commands:
+```
+apt-get install build-essential
+apt-get install libsnappy-dev zlib1g-dev libbz2-dev libgflags-dev liblz4-dev
+git clone https://github.com/facebook/rocksdb.git
```

Comment:
Why do you compile rocksdb from sources? We already have package on our repo. Just add into /etc/apt/sources.list string like:
`deb https://repo.sovrin.org/deb xenial master`
and
`apt install rocksdb`

Reply:
That is a much nicer solution! Im new to this, sorry for the inconvenience.